fastlane/lib/fastlane/actions/setup_circleci.rb

```diff
@@ -3,7 +3,7 @@ module Actions
     class SetupCircleCIAction < Action
       def self.run(params)
         unless should_run?(params)
-          UI.message "Not running on CI, skipping `setup_circleci`"
+          UI.message "Not running on CI, skipping `setup_circle_ci`"
```

Comment:
Was that the name conflict? Do you want the action to be called `setup_circle` or `setup_circle_ci`?

Reply:
nope, the issue is for docs, `CircleCI` became `circle_ci` and when executing it became `circle_c_i`